opentelemetry-sdk/src/opentelemetry/sdk/environment_variables.py

```diff
@@ -335,6 +335,15 @@
 A scheme of https indicates a secure connection and takes precedence over this configuration setting.
 """
 
+OTEL_EXPORTER_OTLP_METRICS_ENDPOINT = "OTEL_EXPORTER_OTLP_METRICS_ENDPOINT"
+"""
+.. envvar:: OTEL_EXPORTER_OTLP_METRICS_ENDPOINT
+
+The :envvar:`OTEL_EXPORTER_OTLP_METRICS_ENDPOINT` target to which the metrics exporter is going to send metrics.
+The endpoint MUST be a valid URL host, and MAY contain a scheme (http or https), port and path.
+A scheme of https indicates a secure connection and takes precedence over this configuration setting.
```

Comment:
_takes precedence over this configuration setting_

What does "this" refer to here? Does it mean `OTEL_{}_INSECURE` setting?

Reply:
This is just the equivalent documentation with traces. The spec has updated the documentation for this environment variable and others but this is better addressed together in a separate PR.